core/src/main/java/com/netflix/iceberg/BaseMetastoreTables.java

```diff
@@ -26,7 +26,7 @@
 import static com.netflix.iceberg.TableMetadata.newTableMetadata;
 
 public abstract class BaseMetastoreTables implements Tables {
-  private final Configuration conf;
+  protected final Configuration conf;
```

Comment:
Since this is passed through the constructor, I think this should remain private. The subclass can keep a reference to the configuration by saving one if it needs.